src/api/app/assets/stylesheets/webui/application/comments.scss

```diff
@@ -17,9 +17,6 @@
   border-top: none;
 }
 
-.comment-actions {
```

Comment:
The class is still used in app/views/webui/comment/_links.html.erb. Please remove it.